src/main/java/jenkins/plugins/office365connector/Webhook.java

```diff
@@ -62,7 +65,9 @@ public Webhook(String name, String url, boolean startNotification, boolean notif
             this.notifyAborted = notifyAborted;
             this.notifyRepeatedFailure = notifyRepeatedFailure;
             this.timeout = timeout;
-            this.macros = macros;
+            if (CollectionUtils.isNotEmpty(macros)) {
```

Comment:
and this fixes regression - if you had no macros defined then NPE was thrown because collection was not defined - now it is at least empty

Reply:
There is a better option of this from `Hudson.Util` which is `this.macros = Util.fixNull(macros)`
http://javadoc.jenkins.io/hudson/Util.html#fixNull-java.util.Collection-

Reply:
Just this one, can you submit as a separate PR. I will merge it.

Reply:
PR is now able to be merged, I suspect that #21 solve merge conflict.